types/async-lock/index.d.ts

```diff
@@ -82,6 +88,7 @@ declare namespace AsyncLock {
 declare class AsyncLock {
     static readonly DEFAULT_TIMEOUT: number;
     static readonly DEFAULT_MAX_OCCUPATION_TIME: number;
+    static readonly DEFAULT_MAX_EXECUTION_TIME: number;
     static readonly DEFAULT_MAX_PENDING: number;
```

Comment:
I think we can change this to:
```suggestion
    static readonly DEFAULT_TIMEOUT: 0;
    static readonly DEFAULT_MAX_OCCUPATION_TIME: 0;
    static readonly DEFAULT_MAX_EXECUTION_TIME: 0;
    static readonly DEFAULT_MAX_PENDING: 1000;
```